### ISSUE TITLE
chore: Introduce handler function for `config` commands 

### ISSUE THREAD

**Issue by jaideepr97** [instructlab/instructlab#2331]
Is your feature request related to a problem? Please describe.
facilitate client/server de-coupling in ilab CLI

Describe the solution you'd like
Introduce handler functions for all functionality invoked via the`config` command group

Additional context
relates to design change described in https://github.com/instructlab/instructlab/issues/2325

**Comment by cdoern** [comment]
Yep that sounds good @jaideepr97 feel free to assing how you see fit
